packages/editor/src/client/panel/memberPlanEditPanel.tsx

```diff
@@ -238,17 +241,16 @@ export function MemberPlanEditPanel({id, onClose, onSave}: MemberPlanEditPanelPr
             </FormGroup>
             <FormGroup>
               <ControlLabel>{t('memberPlanList.minimumMonthlyAmount')}</ControlLabel>
-              <FormControl
-                name={t('userSubscriptionEdit.minimumMonthlyAmount')}
-                value={amountPerMonthMin}
-                type="number"
-                disabled={isDisabled}
-                min={0}
-                steps={1}
+              <InputNumber
+                prefix="CHF"
+                value={parseFloat(amountInFranken as string).toFixed(2)}
```

Comment:
String -> Float -> String here?

Reply:
`toFixed` returns a string, which is necessary to get 2 decimals, then `parseFloat` makes it a number again

Reply:
You are here actually first making it `float` and then back to `string` with `toFixed`.
I guest this is to add the 2 decimals for full amounts. Ok then.